corehq/apps/couch_sql_migration/tests/test_migration.py

```diff
@@ -59,7 +67,9 @@
     softer_assert,
     trap_extra_setup,
 )
-from six.moves import zip
+from couchforms.models import XFormInstance
```

Comment:
This is already imported in this file (possibly introduced by the conflicting PR).

Also this should go above corehq according to https://github.com/dimagi/commcare-hq/blob/master/.isort.cfg#L7

Reply:
Thanks @emord. I agree with isort.